tests/http/testenv/httpd.py

```diff
@@ -104,13 +104,21 @@ def exists(self):
         return os.path.exists(self._cmd)
 
     def set_extra_config(self, domain: str, lines: Optional[Union[str, List[str]]]):
+        if domain not in self._extra_configs:
```

Comment:
The name `set_extra_confg` was intended to replace. If you need an `add_extra_config` it would be better to make that a new method.